src/translate.rs

```diff
@@ -786,6 +786,77 @@ impl Drop for HashTable {
     }
 }
 
+impl<'a, T> ToGlibContainerFromSlice<'a, *const glib_ffi::GArray> for &'a T
+where T: GlibPtrDefault + ToGlibPtr<'a, <T as GlibPtrDefault>::GlibType> {
+    type Storage = (Option<Array>, Vec<Stash<'a, <T as GlibPtrDefault>::GlibType, &'a T>>);
+
+    #[inline]
+    fn to_glib_none_from_slice(t: &'a [&'a T]) -> (*const glib_ffi::GArray, Self::Storage) {
+        let (list, stash) = ToGlibContainerFromSlice::<*mut glib_ffi::GArray>::to_glib_none_from_slice(t);
+        (list as *const glib_ffi::GArray, stash)
+    }
+
+    #[inline]
+    fn to_glib_container_from_slice(_t: &'a [&'a T]) -> (*const glib_ffi::GArray, Self::Storage) {
+        unimplemented!()
+    }
+
+    #[inline]
+    fn to_glib_full_from_slice(_t: &[&'a T]) -> *const glib_ffi::GArray {
+        unimplemented!()
+    }
+}
+
+pub struct Array(*mut glib_ffi::GArray);
+
+impl Drop for Array {
+    fn drop(&mut self) {
+        unsafe { glib_ffi::g_array_free(self.0, false.to_glib()); }
+    }
+}
+
+impl<'a, T> ToGlibContainerFromSlice<'a, *mut glib_ffi::GArray> for T
```

Comment:
This will only work for storing pointers inside the `GArray`. Other use cases are plain fundamental values and storing `struct`s by value. The latter is what @russel actually needs AFAIU.

Reply:
Basically the difference between `Vec<T>` and `Vec<Box<T>>`

Reply:
Oh I see. I'll update then.